jetleopard/JetLeopardWhitePaper.md

```diff
@@ -46,14 +54,13 @@ The queues are not the standard java.util.concurrent queues and instead use wait
 
 The end result is that Jet provides several different APIs to the programmer:
 
-* Distributed java.util.stream
-* DAG
-* Processor
 * Pipelines
+* Distributed java.util.stream
+* DAG (and Processors)
 
-Of these, the Pipelines API is by far the easiest to program.
-The distributed Stream API has additional complexity that many programmers may find a hinderance.
-The DAG API has existed for longer, but is not necessary (and is much too low-level) for most applications of Jet.
+Of these, the Pipelines API is the easiest to program for the newcomer to distributed computing.
+The distributed Stream API has additional complexity that many programmers may find a hinderance at first.
```

Comment:
I don't quite agree with this statement - What is the additional complexity here? I think it might be good to point to our API comparison matrix from the website here?

Reply:
Distributed systems are different to single JVM, and that can represent a barrier. Maybe "some programmers may find a hinderance at first"? Probably a good idea to link - it's a useful matrix.